apps/sms/services/js/moz_mobile_message/moz_mobile_message_shim.js

```diff
@@ -2,8 +2,8 @@
 /* vim: set shiftwidth=2 tabstop=2 autoindent cindent expandtab: */
```

Comment:
Yeah I'll try to make the desktop mozMobileMessage.retrieveMMS returns Promise in this patch. For the integration test, we'll need separate bugs for it since we don't even have test case for MMS retrieval in integration test now.